skimage/filters/_sparse.py

```diff
@@ -89,11 +89,11 @@ def correlate_sparse(image, kernel, mode='reflect'):
 
     Parameters
     ----------
-    image : ndarray, dtype float, shape (M, N,[ ...,] P)
+    image : ndarray, dtype float, shape (M, N[, ...], P)
```

Comment:
Does this change come from black?

Reply:
No, it is done by me. This placement of brackets is highly prevalent across the codebase.

Reply:
Ok, good to know.